### PR TITLE
MOBEXP-56 Basic table component

### PR DESCRIPTION
**Pull request by chrishylanduk**

### CODE REVIEW

File: app/components/table/headRow/index.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { Row } from "react-native-reanimated-table";
+import { type FC } from "react";
+import React from "react";
```
Comment (Aidan23Smith) [inline app/components/table/headRow/index.tsx:3]: Can this be 
```suggestion
import { type FC, React } from "react";
```
?

File: app/components/table/headRow/styles.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  // },
+  // cell: {
+  //   paddingTop: 10,
+  //   paddingRight: 20,
+  //   paddingBottom: 10,
+  //   paddingLeft: 0,
+  //   borderBottom: "1px solid #b1b4b6",
+  //   textAlign: "left",
+  //   verticalAlign: "top",
+  // },
```
Comment (Aidan23Smith) [inline app/components/table/headRow/styles.tsx:35]: remove this?

File: app/components/table/headRow/styles.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import { StyleSheet } from "react-native";
+
+export const govukHeadRowStyle = StyleSheet.create({
+  head: { height: 40, borderBottomWidth: 1, borderBottomColor: "#b1b4b6" },
+  text: { margin: 6, fontWeight: "700" },
```
Comment (Aidan23Smith) [inline app/components/table/headRow/styles.tsx:5]: Can you use the generic text values?

File: app/components/table/headRow/index.tsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+import type { govukHeadRowType } from "./types";
+import { govukHeadRowStyle } from "./styles";
+
+const GovukHeadRow: FC<govukHeadRowType> = (props) => {
+  const combinedStyle = [govukHeadRowStyle.head, props.style];
+  const combinedTextStyle = [govukHeadRowStyle.text, props.textStyle];
```
Comment (Aidan23Smith) [inline app/components/table/headRow/index.tsx:9]: Is the style just the bold text? Do you need a new style?

File: app/components/table/rows/styles.tsx [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  // },
+  // cell: {
+  //   paddingTop: 10,
+  //   paddingRight: 20,
+  //   paddingBottom: 10,
+  //   paddingLeft: 0,
+  //   borderBottom: "1px solid #b1b4b6",
+  //   textAlign: "left",
+  //   verticalAlign: "top",
+  // },
```
Comment (Aidan23Smith) [inline app/components/table/rows/styles.tsx:35]: Is this going to be used?

File: app/components/table/rows/styles.tsx [hunk 6]
```diff
@@ -0,0 +1,36 @@
+import { StyleSheet } from "react-native";
+
+export const govukRowsStyle = StyleSheet.create({
+  text: { margin: 6 },
+  style: { borderBottomWidth: 1, borderBottomColor: "#b1b4b6" },
```
Comment (Aidan23Smith) [inline app/components/table/rows/styles.tsx:5]: Again, is this different from defaults?

File: app/components/table/table/styles.ts [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+  // },
+  // cell: {
+  //   paddingTop: 10,
+  //   paddingRight: 20,
+  //   paddingBottom: 10,
+  //   paddingLeft: 0,
+  //   borderBottom: "1px solid #b1b4b6",
+  //   textAlign: "left",
+  //   verticalAlign: "top",
+  // },
```
Comment (Aidan23Smith) [inline app/components/table/table/styles.ts:35]: These are everywhere

File: app/components/table/table/styles.ts [hunk 8]
```diff
@@ -0,0 +1,36 @@
+import { StyleSheet } from "react-native";
+
+export const govukTableStyle = StyleSheet.create({
+  border: {},
+  style: { borderBottomWidth: 1, borderBottomColor: "#b1b4b6" },
```
Comment (Aidan23Smith) [inline app/components/table/table/styles.ts:5]: Generic colour?

File: app/components/table/rows/styles.tsx (hunk 6 above)
Comment (chrishylanduk) [reply app/components/table/rows/styles.tsx:5]: Text does need the margin adding (for all 3 components)

**Review by Aidan23Smith — APPROVED** [review]

**Pull request merged**